R/functions-imputation.R

```diff
@@ -52,15 +52,20 @@ imputeRowMin <- function(x, min_fraction = 1/2) {
 #'
 #' @description
 #'
-#' Replace missing values with random numbers from a normal distribution based
+#' Replace missing values with random numbers.  
+#' When using the method **mean_sd**, random numbers will be generated 
```

Comment:
Please write here `method = "mean_sd"` (i.e.  \`method = "mean_sd"\`), so it's easier for the user to understand how they can use this method.